src/components/Modal/Modal.test.tsx

```diff
@@ -99,7 +99,7 @@ describe('Modal', () => {
   it('does not throw an error if modal uses aria-label', () => {
     const modalWithAriaLabel = (
       <Modal
-        ariaLabel="aria label"
+        aria-label="aria label"
```

Comment:
You need to update the `Modal` code because the test is looking for `ariaLabel`, which you're not using anymore https://github.com/chanzuckerberg/edu-design-system/blob/main/src/components/Modal/Modal.tsx#L27